shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/segment/ddl/column/alter/ModifyColumnDefinitionSegment.java

```diff
@@ -39,9 +39,11 @@
     private final int stopIndex;
     
     private final ColumnDefinitionSegment columnDefinition;
+
+    private String previousColumnName = "";
```

Comment:
@arthasking123 Add a `ColumnDefinitionSegment newColumnDefinition` field to handle this case is better.